src/lux.ts

```diff
@@ -1516,6 +1517,7 @@ LUX = (function () {
       nErrors +
       "nt" +
       navigationType() +
+      deliveryType() +
```

Comment:
It's a weird format but this is a string of values, delimited by a two-character key. So this will need to be `"dt" + deliveryType()` so that it can be parsed at beacon termination.